Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Prefabs/Placard.prefab

```diff
@@ -209,20 +209,20 @@ TextMesh:
   m_FontSize: 100
   m_FontStyle: 0
   m_RichText: 1
-  m_Font: {fileID: 12800000, guid: 2a056e2bb89e0134daaf49e5f183e5dc, type: 3}
+  m_Font: {fileID: 12800000, guid: 55651ae5ac8b2804f94b620dd47797c3, type: 3}
   m_Color:
     serializedVersion: 2
     rgba: 4294967295
---- !u!114 &114357578567810392
+--- !u!114 &114283598719554152
 MonoBehaviour:
   m_ObjectHideFlags: 1
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1188282787330574}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: ac8d5b128a1d8204fb76c86f47b75912, type: 3}
```

Comment:
Instead, should we make sure the meta files are the same here as in master? That makes the porting story clearer for files with minimal/no changes, like Billboard and the fonts above.